MdePkg/Include/IndustryStandard/IpmiNetFnSensorEvent.h

```diff
@@ -42,5 +42,37 @@ typedef struct {
   UINT8    OEMEvData3;
 } IPMI_PLATFORM_EVENT_MESSAGE_DATA_REQUEST;
 
+//
+// Definitions for Set Sensor Thresholds command
+//
+#define IPMI_SENSOR_SET_SENSOR_THRESHOLDS  0x26
+
+typedef struct _IPMI_SENSOR_SET_SENSOR_THRESHOLD_REQUEST_DATA {
+  UINT8    SensorNumber;
+  UINT8    SetBitEnable;
+  UINT8    LowerNonCriticalThreshold;
+  UINT8    LowerCriticalThreshold;
+  UINT8    LowerNonRecoverablethreshold;
+  UINT8    UpperNonCriticalThreshold;
+  UINT8    UpperCriticalThreshold;
+  UINT8    UpperNonRecoverablethreshold;
+} IPMI_SENSOR_SET_SENSOR_THRESHOLD_REQUEST_DATA;
+
+//
+// Definitions for Get Sensor Thresholds command
+//
+#define IPMI_SENSOR_GET_SENSOR_THRESHOLDS  0x27
+
+typedef struct _IPMI_SENSOR_GET_SENSOR_THRESHOLD_RESPONSE_DATA {
+  UINT8    CompletionCode;
+  UINT8    SetBitEnable;

```

Comment:
Should this be GetBitEnable in this case since its the read not the write?

Reply:
Yes. Dependin on the response to https://github.com/microsoft/mu_basecore/pull/288/files#r1120671605, I will either change it to GetBitEnable, or I will change it to the resulting union.